src/app/store/api/actions.js

```diff
@@ -51,7 +51,6 @@ const getMessage = (error: Error & { response: ?Object }) => {
 const showError = error => dispatch => {
   const errorMessage = getMessage(error);
   dispatch(notificationActions.addNotification(errorMessage, 'error'));
-  error.apiMessage = errorMessage;
```

Comment:
We need to keep this one for error reporting later on.

Reply:
Restored.